examples/transforms/scale.rs

```diff
@@ -13,7 +13,7 @@ struct Scaling {
     min_element_size: f32,
 }
 
-// Implement a simple initialisation.
+// Implement a simple initialization.
```

Comment:
That's a UK vs US English thing, but I agree we should use “-zation” instead of “-sation”: it's more common internationally.